guides/common/attributes-satellite.adoc

```diff
@@ -25,7 +25,8 @@
 :UpdatingDocURL: {BaseURL}updating_red_hat_satellite/index#
 :UpgradingDocURL: {BaseURL}upgrading_connected_red_hat_satellite_to_{ProjectVersion}/index#
 :UpgradingDisconnectedDocURL: {BaseURL}upgrading_disconnected_red_hat_satellite_to_{ProjectVersion}/index#
-:UpgradingPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_red_hat_satellite_to_{ProjectVersionPrevious}/index#
+:UpgradingConnectedPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_connected_red_hat_satellite_to_{ProjectVersionPrevious}/index#
+:UpgradingDisconnectedPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_disconnected_red_hat_satellite_to_{ProjectVersionPrevious}/index#
```

Comment:
You can use the RHBaseURL now, can't you? :)
+ remove the word Connected from the attribute name for consistency

Reply:
Yes :+1: